src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
```

Comment:
Not really responsive, if it's a requirement should be changed to 21-23% (but since you use a border box, 25% is a good choice), in this case, the .container should have justify-content: space-between prop